signup-service/src/main/java/io/mosip/signup/services/IdentityVerificationService.java

```diff
@@ -147,27 +164,38 @@ public SlotResponse getSlot(String transactionId, SlotRequest slotRequest) {
 
     private String fetchAndVerifyAccessToken(String authCode) {
         try {
-            AuthorizationCode code = new AuthorizationCode(authCode);
-            URI callback = new URI(oauthRedirectUri);
-            AuthorizationGrant codeGrant = new AuthorizationCodeGrant(code, callback);
-
             long issuedTime = System.currentTimeMillis();
             JWTClaimsSet.Builder builder = new JWTClaimsSet.Builder()
                     .subject(oauthClientId)
                     .issuer(oauthClientId)
-                    .audience(oauthIssuerUri)
+                    .audience(oauthIssuerUri+"/v1/esignet/oauth/token")
```

Comment:
Can we take this value from configuration? 
mosip.signup.oauth.audience

Reply:
done